src/index.js

```diff
@@ -61,10 +63,74 @@ const updateTask = () => {
     list[i].addEventListener('change', () => {
       collection.array[i].description = list[i].value;
       const stringData = JSON.stringify(collection.array);
-      localStorage.setItem('tasks', stringData);
+      localStorage.setItem('Tasks', stringData);
       window.location.reload();
     });
   }
 };
 updateTask();
-export default collection;
\ No newline at end of file
+
+// Add functionality to checkbox
+for (let i = 0; i < collection.array.length; i += 1) {
+  const checkbox = document.getElementById(`box${i}`);
+  checkbox.addEventListener('change', () => {
+    if (collection.array[i].complete === false) {
+      collection.array[i].complete = true;
+      localStorage.setItem('Tasks', JSON.stringify(collection.array));
+
+      const list = document.getElementById(`list${i}`);
+      list.style.textDecoration = 'line-through';
+    } else if (collection.array[i].complete === true) {
+      collection.array[i].complete = false;
+      localStorage.setItem('Tasks', JSON.stringify(collection.array));
+
+      const lists = document.getElementById(`list${i}`);
+      lists.style.textDecoration = 'none';
+    }
+  });
+}
```

Comment:
- [ ] The user should get the correct list values after the page reloads. Please make sure when you reload the page the checkboxes next to the tasks are checked if the tasks are completed.